Silksong.ModMenu.csproj

```diff
@@ -54,8 +82,8 @@
     </PropertyGroup>
     <Copy
       SourceFiles="@(Binaries)"
-      DestinationFolder="$(SilksongFolder)/BepInEx/plugins/$(TargetName)"
-      Condition="'$(SilksongFolder)' != '' And Exists('$(SilksongFolder)')"
+      DestinationFolder="$(SilksongPluginsDir)/BepInEx/plugins/$(TargetName)"
+      Condition="'$(SilksongPluginsDir)' != '' And Exists('$(SilksongPluginsDir)')"
```

Comment:
I'm guessing something weird is going on here, SilksongPluginsDir seems to have BepInEx/plugins earlier but not here

Reply:
So I do want two separate variables for 'plugins dir' and 'install dir' in order to be able to link against the real source code when developing locally while also using r2modman for plugin management.  I'm not sure which one of those should be reified to 'SilksongFolder', I guess the plugins one?